packages/next/client/link.js

```diff
@@ -42,16 +42,23 @@ class Link extends Component {
     }
   }
 
+  formatUrl (url, asHref) {
```

Comment:
Since this method only uses static methods I'd move it out of the class

Reply:
Good point, updated